compose.yml

```diff
@@ -2,9 +2,6 @@ version: "3.8"
 services:
   # python-matter-server
   matter-server:
-    build:
-      context: ./
-      dockerfile: Dockerfile.dev
```

Comment:
do we also need to delete the dockerfile.dev ?

Reply:
I've converted the `compose.yml` to use the new container.

However, for development the container *could* still be useful. On the other hand, probably a dev container would be more appropriate then. Thoughts? I am fine removing it.

Reply:
I think we should keep it until we have an alternative in devcontainer.